config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: ocs-dev/ocs-operator
-  newName: quay.io/ocs-dev/ocs-operator
+  newName: quay.io/pkundra/ocs-operator
```

Comment:
I don't think that this should be merged in the official code.
This PR should probably be marked Draft or WIP until this is resolved.

Reply:
testing of the patch is pending. The test are failing.